user/src/com/google/gwt/user/server/rpc/RemoteServiceServlet.java

```diff
@@ -280,7 +283,9 @@ public final SerializationPolicy getSerializationPolicy(String moduleBaseURL,
    * @param payload the UTF-8 request payload
    * @return a string which encodes either the method's return, a checked
    *         exception thrown by the method, or an
-   *         {@link IncompatibleRemoteServiceException}
+   *         {@link IncompatibleRemoteServiceException}, or {@code null} if a protocol version of
```

Comment:
I'm surprised that the exception is in the `@return` - could you bump that down to its own `@throws`, so we only see null vs value?

Reply:
3f684820095fafd80ed35a6ab9f47d448505ea6f